src/style.css

```diff
@@ -1 +1,66 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.page {
+  margin: 0;
+  padding-top: 20px;
+}
+
+.searchbars {
+  margin: 0 4px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.searchbars[data-qa="big"] input {
```

Comment:
Use selectors by class, it's much better and fits better with the bem methodology for which it was created.